src/porting.cpp

```diff
@@ -511,6 +512,12 @@ void initializePaths()
 		errorstream << "Failed to get one or more system-wide path" << std::endl;
 
 #endif
+#ifdef STATIC_LOCALEDIR
+	if (STATIC_LOCALEDIR[0] != '\0')
+		path_locale = STATIC_LOCALEDIR;
+	else
```

Comment:
That would break it, the later line would reset `path_locale`.
Is this better?
```C++
#ifdef STATIC_LOCALEDIR
	path_locale = STATIC_LOCALEDIR[0] ? STATIC_LOCALEDIR : getDataPath("locale");
#else
	path_locale = getDataPath("locale");
#endif
```